pyttb/gcp/handles.py

```diff
@@ -84,17 +84,17 @@ def rayleigh(data: np.ndarray, model: np.ndarray) -> np.ndarray:
 
 def rayleigh_grad(data: np.ndarray, model: np.ndarray) -> np.ndarray:
     """Return gradient function for rayleigh distributions"""
-    return 2 / (model + EPS) - (np.pi / 2) * data**2 / (model + EPS) ** 3
+    return 2 / (model + EPS) - (np.pi / 2) * np.power(data, 2) / (model + EPS) ** 3
 
 
 def gamma(data: np.ndarray, model: np.ndarray) -> np.ndarray:
     """Return objective function for gamma distributions"""
-    return data / (model + EPS) + np.log(model + EPS)
+    return np.multiply(data,-1) / (model + EPS) + np.log(model + EPS)
```

Comment:
Hmm am I missing something here? It looks like in MATLAB the negative is on the gamma and not the grad. https://gitlab.com/tensors/tensor_toolbox/-/blob/dev/tt_gcp_fg_setup.m#L83 is that a bug in MATLAB as well then?